forc-lsp/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "forc-lsp"
+version = "0.9.2"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/sway"
+description = "A simple `forc` plugin for starting the sway language server."
+
+[dependencies]
+anyhow = "1"
+clap = { version = "3", features = ["derive"] }
+sway-lsp = { version = "0.9", path = "../sway-lsp" }
```

Comment:
Wait no, Fuel deps still need an exact version otherwise the publishing action won't work :joy:

Reply:
agh of course!

Reply:
We really need to augment https://github.com/FuelLabs/cargo-toml-lint to check for that!